### ISSUE TITLE
_feeds.buzzsprout.com_ feeds give error

### ISSUE THREAD

**Issue by Jackie7610** [XilinJia/Podcini#141]
Greetings dev. Merry christmas🥳🍻 _(hope Im not too late)_

I've noticed that any podcast feed starting with `feeds.buzzsprout.com` will give an error and will not show any episodes. This error is not shown in Antennapod or other 3rd party podcast apps. Could you please fix it?

Podcini version: 7.1.1 _(installed from github releases)_

https://imgur.com/a/WsHtZGy

Some podcasts which use buzzsprout

1. Written in Red (by Aron Beauregard)

2. Night Walks (Robyn August)

3. The Exit Light Podcast (by Eli)

4. Writing with Trauma (by William Thompson)

5. Sadistic (by Jon Saks)

**Comment by XilinJia** [comment]
OK. Thanks for reporting. 

Caused by redirect ability added in 7.1.0.  It could affect many podcasts.  fixed in the coming release.

**Comment by XilinJia** [comment]
check out 7.2.0

**Comment by Jackie7610** [comment]
Yep. Thanks dev. It's working now, no more errors. Only the podcast art is replaced with the podcini icon. 

But since it's working...I can close this issue?

**Comment by XilinJia** [comment]
> the podcast art is replaced with the podcini icon

this happens when the related podcast image is unavailable for some reason.  If you see more cases like this, let me know.

Feel free to close this unless you have additional problems with this.

**Comment by Jackie7610** [comment]
The podcast art can wait I guess till it gets back to normal. I'll close this issue. 
Thanks again. 😃
